components/script/dom/touchevent.rs

```diff
@@ -92,6 +108,7 @@ impl TouchEvent {
             view,
             detail,
         );
+        ev.upcast::<Event>().set_composed(bool::from(composed));
```

Comment:
```suggestion
        ev.upcast::<Event>().set_composed(composed.into());
```
Just to be consistent.